### DISCUSSION TITLE
Road to have `iso` build

### DISCUSSION THREAD

**Discussion by mkikets99** [krglaws/MyLFS#10] [Ideas]
I will go for it after `Multi-phased` road is done.

TODO (Will be updated):
- [ ] Check if `syslinux` files is generated
- [ ] Prepare files to generate `iso-fs`
- [ ] check if grub is generated for both BOOT and UEFI
- [ ] generate `iso` with `xorriso`

**Comment by mkikets99** [comment]
About Grub from #5 
It is building... when `syslinux` is on the list the `grub` line pops up... so it is an output issue

**Reply by krglaws** [reply]
I'm not sure, I just ran a full build yesterday, no "Building grub..." line, and no grub build log. It might be skipping whatever is on the last line for some reason. But it does build if I tell it to go straight to the grub build with `--start-phase 4 --start-package grub`.

**Reply by mkikets99** [reply]
Oh... i found something...
So.. if run with `--build-all` it skips all after `linux` script...
But it's not when running with `--start-phase ...` section
Not sure if it is good when running from `1-3` section... but its quite interesting...
